docs/scoring.md

```diff
@@ -0,0 +1,39 @@
+# Goal
+
+The goal of this document is to explain how scoring on Lighthouse works, and how to understand what to do to improve your
+Lighthouse score. 
+
+# What metrics does Lighthouse measure?
+
+Lighthouse primarily measures the following metrics: 
+
+- First meaningful paint: first meaningful paint is defined as when the browser first puts any “meaningful” element/set of “meaningful” elements  on the screen. What is meaningful is determined from a series of heuristics. 
+- First interactive: first interactive is defined as the first point at which the page could respond quickly to input. It doesn't consider any point in time before first meaningful paint. The way this is implemented is primarily based on heuristics. 
+*Note: this metric is currently in beta, which means that the underlying definition of this metric is in progress.*
+- Consistently interactive: defined as that everything is loaded such that the page will quickly respond to any user input throughout the page. 
+*Note: this metric is currently in beta, which means that the underlying definition of this metric is in progress.*
+- Perceptual Speed Index (pSI): pSI measures how many pixels are painted at each given time interval on the viewport. The fewer the pixels that are painted, the better since we want an experience where most of the content is shown on the screen during the first few moments of initiating the page load. Therefore, the lower the pSI score, the better. 
+- Estimated Input Latency:  this audit measures how fast your app is in responding to user input. Our benchmark is that the estimated input latency should be under 50 ms (see documentation here as to why)
+
+*Note that by Lighthouse can experience weird behaviors when it run in the presence of anti-virus scanners. Please try to run without the anti-virus scanner to get the cleanest results.*
+
+# How are the scores weighted?
+Lighthouse returns a performance score from 0-100. A score of 0 usually indicates an error with performance measurement (so file an issue in the Lighthouse repo if further debugging is needed), and 100 is the best possible ideal score (really hard to get). 
+We usually recommend that if developers are scoring around a 95, their websites are extremely performant. Any additional improvements to the score past 95 doesn’t add additional value for the developer, so we don’t recommend they invest further time to improve the score. 
```

Comment:
see other comment below, perhaps a little stronger than we want to say to devs :)

Reply:
deleted this section.